packages/babel-preset-react-app/create.js

```diff
@@ -96,9 +107,12 @@ module.exports = function(api, opts, env) {
           // Adds component stack to warning messages
           // Adds __self attribute to JSX which React will use for some warnings
           development: isEnvDevelopment || isEnvTest,
-          // Will use the native built-in instead of trying to polyfill
+          // Use native spread when possible.
+          // Or use the native built-in instead of trying to polyfill
           // behavior for any plugins that require one.
-          useBuiltIns: true,
+          [isObjectRestSpreadProposalRequired
+            ? 'useBuiltIns'
+            : 'useSpread']: true,
```

Comment:
Here we need to use native spread whenever possible. This requires knowing if our targets require 'proposal-object-rest-spread'.